tests/services/test_service_users.py

```diff
@@ -12,6 +12,20 @@
 from invenio_access.permissions import system_identity
 from invenio_records_resources.services.errors import PermissionDeniedError
 
+from invenio_users_resources.proxies import current_actions_registry
+
+
+@pytest.fixture(scope="function", autouse=True)
+def mock_action_registry(monkeypatch, user_service):
+    """Mocks action to registry entirely.
+
+    Monkeypatches the registry to be modified only for testing.
+    """
+
+    monkeypatch.setitem(current_actions_registry, "block", [])
```

Comment:
The idea is that after actions are not set (entry points load them) unless explicitly specified by a test.